### PR TITLE
Header footer issues

### PR DESCRIPTION
**Pull request by BahadurAliShah**

### CODE REVIEW

File: src/components/Header.jsx [hunk 1]
```diff
@@ -100,10 +100,10 @@ export function Header() {
                         >
                           <div className="space-y-4">
                             <Services/>
-                            <MobileNavLink href="#pricing">
+                            <MobileNavLink href="pricing">
                               Pricing
                             </MobileNavLink>
-                            <MobileNavLink href="#reviews">
+                            <MobileNavLink href="#contact">
```
Comment (MZO9400) [inline src/components/Header.jsx:106]: Why are we not linking the HTML pages?

File: src/components/Header.jsx [hunk 2]
```diff
@@ -100,10 +100,10 @@ export function Header() {
                         >
                           <div className="space-y-4">
                             <Services/>
-                            <MobileNavLink href="#pricing">
+                            <MobileNavLink href="pricing">
```
Comment (MZO9400) [inline src/components/Header.jsx:103]: Why are we not linking the HTML pages?

**Review by MZO9400 — CHANGES_REQUESTED** [review]

**Review by hamzafer — APPROVED** [review]

**Comment by hamzafer** [comment]
Please resolve Merge Conflicts.

**Comment by hamzafer** [comment]
Merge conflicts resolved.

File: src/components/Header.jsx (hunk 1 above)
Comment (hamzafer) [reply src/components/Header.jsx:106]: let's merge it for now.
this PR is nothing significant in it.

**Pull request merged**